### ISSUE TITLE
Bind builds should include support for statistics in json format

### ISSUE THREAD

**Issue by bahamat** [TritonDataCenter/pkgsrc-legacy#242]
The bind statistics server supports output in xml (the default) and json by linking against `libxml2` or `json-c` respectively.

The following patch adds (but disabled by default) support for json output in the statistics server.

``` diff
diff --git a/mk/defaults/options.description b/mk/defaults/options.description
index 09b92c9..8460055 100644
--- a/mk/defaults/options.description
+++ b/mk/defaults/options.description
@@ -54,7 +54,8 @@ bacula-static         Enable building of statically linked versions of the Bacula daemo
 batch                  Enable batch mode support.
 bdb                    Enable Berkeley DB support.
 bind-dig-sigchase      Enable dig(1) option +sigchase for DNSSEC signature chasing.
-bind-xml-statistics-server     Enable building in the statistics server in bind95 and later.
+bind-xml-statistics-server     Enable building in the statistics server with XML output in bind95 and later.
+bind-json-statistics-server    Enable building in the statistics server with JSON in bind95 and later.
 bittorrent             Enable bittorrent support.
 blender-exppython      Enable the Experimental Python support.
 boehm-gc               Use Boehm's garbage collector for memory allocation.
diff --git a/net/bind910/options.mk b/net/bind910/options.mk
index 4db9eb9..e3ce265 100644
--- a/net/bind910/options.mk
+++ b/net/bind910/options.mk
@@ -3,6 +3,7 @@
 PKG_OPTIONS_VAR=       PKG_OPTIONS.bind910
 PKG_SUPPORTED_OPTIONS= bind-dig-sigchase bind-xml-statistics-server
 PKG_SUPPORTED_OPTIONS+=        inet6 threads readline mysql pgsql ldap dlz-filesystem
+PKG_SUPPORTED_OPTIONS+=        bind-json-statistics-server
 PKG_SUPPORTED_OPTIONS+=        rrl
 PKG_SUGGESTED_OPTIONS+=        readline

@@ -69,6 +70,16 @@ CONFIGURE_ARGS+=     --without-libxml2
 .endif

 ###
+### The statistics server in bind910 and later supports json
+###
+.if !empty(PKG_OPTIONS:Mbind-json-statistics-server)
+.include "../../textproc/json-c/buildlink3.mk"
+CONFIGURE_ARGS+=       --with-libjson
+.else
+CONFIGURE_ARGS+=       --without-libjson
+.endif
+
+###
 ### IPv6 support
 ###
 .if !empty(PKG_OPTIONS:Minet6)
```

**Comment by bahamat** [comment]
This options for enabling this in the Joyent builds didn't go in. The current package version in 2014Q3 (bind-9.10.0pl2nb1) is compiled `'--without-libxml2' '--without-libjson'`.

**Comment by mamash** [comment]
Implemented yesterday in bind-9.10.0pl2nb1.
